src/test/java/com/sap/oss/phosphor/fosstars/model/score/oss/ArtifactReleaseHistoryScoreTest.java

```diff
@@ -149,7 +149,7 @@ public void testTwoMonthOldVersionStable() {
         ArtifactVersions
             .of(version100, version110, version120, version130, version140, version150));
     ScoreValue value = score.calculate(versions);
-    Assert.assertEquals(7.6, value.get(), DELTA);
+    Assert.assertEquals(8.4, value.get(), DELTA);
```

Comment:
Why the score value has changed here? I don't see you updated the implementation of the scoring function.

Reply:
I am checking this.

Reply:
The test started failing on my machine as well. Looks like a bug in the test or in the scoring function. I'd suggest to add `@Ignore` and open an issue for this.

Reply:
@mibo FYI

Reply:
#511